addons/karpenter/update_env.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export CLUSTER_NAME=$(cd ../../terraform/eks-cluster && terraform output -raw cluster_name)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)